src/mudlet-lua/lua/geyser/GeyserMiniConsole.lua

```diff
@@ -12,6 +12,14 @@ Geyser.MiniConsole = Geyser.Window:new({
   name = "MiniConsoleClass",
   wrapAt = 300, })
 
+--- Override reposition to reset autowrap
+function Geyser.MiniConsole:reposition()
+  Geyser.Window.reposition(self)
```

Comment:
Would this be better as `self.parent:reposition()`?

Reply:
I'd probably use `self.parent.reposition(self)`. Your variant binds the parent as `self`.